edge-modules/azure-monitor/src/write_vernum_print_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Can we remove this script? I don't think it's called from anywhere, and it references another script (`get_vernum.sh`) which doesn't seem to be here either...

Reply:
Removed.